src/main/webapp/WEB-INF/view/searchResults.jsp

```diff
@@ -41,7 +42,7 @@
 				<h3 class="title text-truncate">
 					<a class="link" href="${doc.url_link}" data-uri="${doc.url_link}"
 						data-id="${doc.doc_id}" data-order="${s.index}">${doc.content_title}</a>
-				</h3>
+				</h3>									
```

Comment:
Please remove this change.

Reply:
Sorry for the unnecessary correction.
I removed it.